### ISSUE TITLE
Mars Weather

### ISSUE THREAD

**Issue by PranavBarthwal** [PranavBarthwal/cosmoXplore#5]
https://youtu.be/MDENx86nK4k?list=PLZlA0Gpn_vH8DWL14Wud_m8NeNNbYKOkj

**Comment by SpaciousCoder78** [comment]
Can you assign this task to me?

**Comment by PranavBarthwal** [comment]
@SpaciousCoder78 Thanks for showing interest in this issue.
I'll surely assign this to you after 19:00 so that it gets counted under GSSoC'24.

**Comment by SpaciousCoder78** [comment]
Thanks, I will start working on it.

**Comment by SpaciousCoder78** [comment]
I'm starting the work on the issue.

@PranavBarthwal is there a way I can chat with you? Discord or twitter? This is a big task so I'm gonna need your guidance

**Comment by PranavBarthwal** [comment]
Ping me on twitter, You can find my socials on my Github profile.

**Comment by PranavBarthwal** [comment]
@SpaciousCoder78 
Create this feature in a new page, having same design language and color palette as the home page.
Make sure to create a new folder named `pages` inside that create another folder named `mars-weather` where you can create its html, css and js files.
Happy Coding.

**Comment by SpaciousCoder78** [comment]
Understood. I will start the work and design it accordingly.

**Comment by SpaciousCoder78** [comment]
Unfortunately InSight Mission went offline about a year ago so the API is down too so this feature cannot be worked on anymore.

@PranavBarthwal I request you to assign me to another issue. I already worked on this issue a lot so I have the code of it which can be repurposed and used for other pages.

**Comment by PranavBarthwal** [comment]
You can add a solar system model using Particle.js

**Comment by SpaciousCoder78** [comment]
Can I take up another issue? Solar System Model is not my expertise because I haven't worked on anything like it before.

**Comment by PranavBarthwal** [comment]
> Can I take up another issue? Solar System Model is not my expertise because I haven't worked on anything like it before.

yeah you can find any other issue which is not assigned to anyone.

**Comment by Helios-snd** [comment]
![Screenshot 2024-05-13 010320](https://github.com/PranavBarthwal/cosmoXplore/assets/138819512/bb19571a-b93a-488a-88bb-f5799a6778d0)

If you want something like this? I can do it for you

**Comment by SpaciousCoder78** [comment]
I have unassigned myself from this issue. Please assign this issue to @Helios-snd since they have a solution to this issue.

**Comment by PranavBarthwal** [comment]
> ![Screenshot 2024-05-13 010320](https://private-user-images.githubusercontent.com/138819512/329875766-bb19571a-b93a-488a-88bb-f5799a6778d0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU2MjUyMjgsIm5iZiI6MTcxNTYyNDkyOCwicGF0aCI6Ii8xMzg4MTk1MTIvMzI5ODc1NzY2LWJiMTk1NzFhLWI5M2EtNDg4YS04OGJiLWY1Nzk5YTY3NzhkMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxM1QxODI4NDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMWZjNGY5YjQ3NjJlYjEyOTIyZDUzMTQ5OTY1MzZiNWU0YTM4ZTZhM2QxODAxYzUyYTEwMWMxZDViNmVkYTFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.pgEZy3DCRbsa5QmjGOaDxHrdYXacKFQB6LzDW5Zv8Oc)
> 
> If you want something like this? I can do it for you

Someting like in this project :  https://github.com/02shreyansh/SpaceEdu

**Comment by thecloudcode** [comment]
Hi, @PranavBarthwal , would would something like this for a Solar System Model work?

![image](https://github.com/PranavBarthwal/cosmoXplore/assets/114615639/28e992f0-e7c2-4de4-9b49-9a1c9d7e8d50)

If yes, kindly assign me the issue, and I would love to work on it
